tests/commands/json/map/test_cached_map_info.py

```diff
@@ -13,16 +16,22 @@
 from tests.commands.json import assert_command
 from tests.helpers import get_request_json, get_success_body
 
+if TYPE_CHECKING:
+    from deebot_client.command import Command
+
 
 @pytest.mark.parametrize(
-    ("device_class", "map_set_type"),
+    ("device_class", "map_set_type", "additional_commands"),
     [
-        ("yna5xi", GetMapSet),
-        ("kr0277", GetMapSetV2),
+        ("yna5xi", GetMapSet, []),
+        ("kr0277", GetMapSetV2, []),
+        ("lwmdoj", GetMapSetV2, [GetMapInfoV2]),
```

Comment:
```suggestion
    ("device_class", "map_set_type", "map_info_command"),
    [
        ("yna5xi", GetMapSet, None),
        ("kr0277", GetMapSetV2, None),
        ("lwmdoj", GetMapSetV2, GetMapInfoV2),
```

The list will only make things complex and also the first argument needs to be the map_id

Reply:
Removed the `additional_commands` and now rely on the capabilities for whether `GetMapInfoV2` is expected or not. Hope that's better.

If I'm already loading the capabilities of the device, should I also get from there whether `GetMapSet` or `GetMapSetV2` is expected?